packages/lib/bookingSuccessRedirect.ts

```diff
@@ -35,11 +35,55 @@ export const getBookingRedirectExtraParams = (booking: SuccessRedirectBookingTyp
     "startTime",
     "endTime",
     "location",
+    "attendees",
+    "user",
   ];
 
-  return (Object.keys(booking) as BookingResponseKey[])
+  type ResultType = {
+    [key in BookingResponseKey]?: SuccessRedirectBookingType[key];
+  } & {
+    hostName?: string[];
+    attendeeName?: string | null;
+  };
+
+  const result = (Object.keys(booking) as BookingResponseKey[])
     .filter((key) => redirectQueryParamKeys.includes(key))
-    .reduce((obj, key) => ({ ...obj, [key]: booking[key] }), {});
+    .reduce<ResultType>((obj, key) => {
+      if (key === "user" && booking.user?.name) {
+        return { ...obj, hostName: [...(obj.hostName || []), booking.user.name] };
+      }
+
+      if (key === "attendees" && Array.isArray(booking.attendees)) {
+        const attendeeName = booking.attendees[0]?.name || null;
+        const hostNames = booking.attendees.slice(1).map((attendee) => attendee.name);
```

Comment:
There could be guests as well in it 
![image](https://github.com/user-attachments/assets/6e387ac7-9ceb-43cb-b047-334e958da08a)


I think we should have had a function to get hosts, guests, booker from the booking that could help us avoiding being detective everytime.

Reply:
Yeah I agree. Will line up a follow up on this with these functions in place. Adding guest consideration for now!

Reply:
Added Guest consideration. Will do a follow up next week to improve this with reusable functions to get hosts, guests, etc. which isn't opinionated.